idea/idea-frontend-fir/idea-fir-low-level-api/src/org/jetbrains/kotlin/idea/fir/low/level/api/util/declarationUtils.kt

```diff
@@ -124,11 +124,11 @@ private fun KtClassOrObject.findFir(firSymbolProvider: FirSymbolProvider): FirRe
     }
 }
 
-private fun KtTypeAlias.findFir(firSymbolProvider: FirSymbolProvider): FirTypeAlias {
+private fun KtTypeAlias.findFir(firSymbolProvider: FirSymbolProvider): FirTypeAlias? {
     val typeAlias = ClassId(containingKtFile.packageFqName, nameAsSafeName)
     return executeWithoutPCE {
+        // Finding a non-top-level type alias won't work
```

Comment:
Note that `ClassId` above is created with the package name, and that's why non top-level type alias won't work. If the containing class is given, I wonder if we can still try building a correct class id and sending a lookup based on that.

Reply:
Yeah. But it still won't work for local type alias right? So I didn't try to fix it there. Anyway, since non-toplevel type alias is not supported. Maybe it's not worth spending time trying to resolve them?